vm/src/vm.rs

```diff
@@ -42,6 +43,23 @@ use num_bigint::BigInt;
 #[cfg(feature = "rustpython-compiler")]
 use rustpython_compiler::{compile, error::CompileError};
 
+// Signal triggers
+// TODO: 64
+pub const NSIG: usize = 10;
+
+pub static mut TRIGGERS: [AtomicBool; NSIG] = [
```

Comment:
Maybe you could use `Default::default()` here?

Reply:
Did not work. I will create a procedural macro in the future to solve this duplication.